### PR TITLE
Guide to serial communication

### LINKED ISSUE #8: Serial port, how does it work?
How to print over serial etc

### PR DESCRIPTION
**Pull request by stianjensen**
Linux guide and windows screenshots missing

Fixes #8

### CODE REVIEW

File: guides/serial.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+layout: guide
+banner: images/coolterm-banner.png
```
Comment (sigvef) [inline guides/serial.md:3]: can you make a banner that doesn't have white at the bottom? (where does the image end, and where does the guide start?!)

image

**Comment by stianjensen** [comment]
:dango:

File: guides/serial.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+## Windows
+
+Download the [serial driver](http://developer.mbed.org/media/downloads/drivers/mbedWinSerial_16466.exe).
+
+Plug the mbed into your computer, and then run the installer.
+
+Download [termite](http://www.compuphase.com/software/termite-3.1.exe) which will be used for displaying the print output from the mbed.
+
+## OS X
+
```
Comment (cristeahub) [inline guides/serial.md:39]: Add some `plug mbed into computer and do some setup` stuff here?

Comment (stianjensen) [inline guides/serial.md:39]: What kind of setup?

Comment (cristeahub) [inline guides/serial.md:39]: On the windows version it says you need the mbed plugged in. It might seem obvious, but might not be

**Comment by stianjensen** [comment]
plingplong

**Pull request merged**